src/actions/getRecordingInfo.ts

```diff
@@ -6,8 +6,7 @@ export const getRecordingInfo = async (a: {recordingObject: any, hither: HitherC
         'createjob_get_recording_info',
         { recording_object: a.recordingObject },
         {
-            useClientCache: true,
-            newHitherJobMethod: true
+            useClientCache: true
```

Comment:
Simplify hither calls throughout. The only option on the client (javascript) side is useClientCache which specifies whether we cache the result in the browser memory.